components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -513,6 +523,20 @@ - (void)setItemTitleFont:(UIFont *)itemTitleFont {
   }
 }
 
+- (void)setBarTintColor:(UIColor *)barTintColor {
+  _barTintColor = barTintColor;
+  self.backgroundColor = barTintColor;
+}
+
+- (void)setBackgroundColor:(UIColor *)backgroundColor {
+  super.backgroundColor = backgroundColor;
+  _barTintColor = backgroundColor;
```

Comment:
Does this behavior match UITabBar / UINavigationBar?

Reply:
For both UITabBar and UINavigationBar, setting the `barTintColor` property changes the color of the background of the view.  

## UIKit

**UITabBar.h**
```objectivec
/*
 The behavior of tintColor for bars has changed on iOS 7.0. It no longer affects the bar's background
 and behaves as described for the tintColor property added to UIView.
 To tint the bar's background, please use -barTintColor.
 */
@property(nullable, nonatomic, strong) UIColor *barTintColor NS_AVAILABLE_IOS(7_0) UI_APPEARANCE_SELECTOR;  // default is nil
```

**UINavigationBar.h**
```objectivec
/*
 The behavior of tintColor for bars has changed on iOS 7.0. It no longer affects the bar's background
 and behaves as described for the tintColor property added to UIView.
 To tint the bar's background, please use -barTintColor.
 */
@property(nullable, nonatomic,strong) UIColor *barTintColor NS_AVAILABLE_IOS(7_0) UI_APPEARANCE_SELECTOR;  // default is nil
```

## MDC-iOS

**MDCTabBar.h**

```objectivec
/**
 Tint color to apply to the tab bar background.

 If nil, the receiver uses the default background appearance. Default: nil.
 */
@property(nonatomic, nullable) UIColor *barTintColor UI_APPEARANCE_SELECTOR;
```